doc/schemas/options_schema.json

```diff
@@ -48,12 +48,17 @@
       "additionalProperties": false,
       "properties": {
         "Resolution": {
-          "type": "integer",
+          "type": "array",
```

Comment:
That's definitely an idea we should consider, but I think it should be a separate issue/PR. I forgot to update the schema in #91, and this PR is just fixes that. Changing the options format would also mean updating the `TileFormat` codec and example options files.